images/vertical-pod-autoscaler-operator.yml

```
@@ -22,3 +22,6 @@ from:
 name: openshift/ose-vertical-pod-autoscaler-operator
 owners:
 - joesmith@redhat.com
+update-csv:
+  manifests-dir: manifests/
```

For the bundle format we're going to want this line:
```suggestion
  manifests-dir: manifests/
  bundle-dir: "{MAJOR}.{MINOR}/"
```